dynatrace/environment_v2/problems.py

```diff
@@ -0,0 +1,301 @@
+        self.evidence_details: Optional[EvidenceDetails] = EvidenceDetails(raw_element=raw_element.get("evidenceDetails"))
+        self.impact_analysis: Optional[ImpactAnalysis] = ImpactAnalysis(raw_element=raw_element.get("impactAnalysis"))
+        self.entity_tags: Optional[List[METag]] = [METag(raw_element=t) for t in raw_element.get("entityTags", [])]
+
+
+class ProblemCloseResult(DynatraceObject):
+    def _create_from_raw_data(self, raw_element: Dict[str, Any]):
+        self.problem_id: str = raw_element.get("problemId")
+        self.closing: bool = raw_element.get("closing")
+        self.close_timestamp: datetime = int64_to_datetime(raw_element.get("closeTimestamp"))
+        self.comment: Optional[Comment] = Comment(raw_element=raw_element.get("comment"))
+
+
+class LinkedProblem(DynatraceObject):
+    def _create_from_raw_data(self, raw_element: Dict[str, Any]):
+        self.display_id: str = raw_element.get("displayId")
+        self.problem_id: str = raw_element.get("problemId")
+
+
+class CommentList(DynatraceObject):
+    def _create_from_raw_data(self, raw_element: Dict[str, Any]):
+        self.total_count: int = raw_element.get("totalCount")
+        self.comments: Optional[List[Comment]] = [Comment(raw_element=c) for c in raw_element.get("comments", [])]
+
+
+class Comment(DynatraceObject):
+    def _create_from_raw_data(self, raw_element: Dict[str, Any]):
+        self.created_at: datetime = int64_to_datetime(raw_element.get("createdAtTimestamp"))
+        self.author: Optional[str] = raw_element.get("authorName")
+        self.context: Optional[str] = raw_element.get("context")
+        self.id: Optional[str] = raw_element.get("id")
+        self.content: Optional[str] = raw_element.get("content")
+
+
+class EvidenceDetails(DynatraceObject):
+    def _create_from_raw_data(self, raw_element: Dict[str, Any]):
+        self.total_count: int = raw_element.get("totalCount")
+        self.details: Optional[List[Evidence]] = (
+            [EventEvidence(raw_element=e) for e in raw_element.get("details", []) if e.get("evidenceType") == EvidenceType.EVENT]
+            + [MetricEvidence(raw_element=e) for e in raw_element.get("details", []) if e.get("evidenceType") == EvidenceType.METRIC]
+            + [TransactionalEvidence(raw_element=e) for e in raw_element.get("details", []) if e.get("evidenceType") == EvidenceType.TRANSACTIONAL]
+            + [MaintenanceWindowEvidence(raw_element=e) for e in raw_element.get("details", []) if e.get("evidenceType") == EvidenceType.MAINTENANCE_WINDOW]
+            + [AvailabilityEvidence(raw_element=e) for e in raw_element.get("details", []) if e.get("evidenceType") == EvidenceType.AVAILABILITY]
+        )
+
+
+class Evidence(DynatraceObject):
+    def _create_from_raw_data(self, raw_element: Dict[str, Any]):
+        self.evidence_type: EvidenceType = EvidenceType(raw_element=raw_element.get("evidenceType"))
```

Comment:
This is an `Enum`, not a `DynatraceObject` so it does not have a `raw_element` kwarg, I believe this should be

 `EvidenceType(raw_element.get("evidenceType"))`